packages/visx-scale/package.json

```diff
@@ -33,10 +33,10 @@
   "dependencies": {
     "@types/d3-interpolate": "^1.3.1",
     "@types/d3-scale": "^3.2.1",
-    "@types/d3-time": "^1.0.10",
+    "@types/d3-time": "^2.0.0",
     "d3-interpolate": "^1.4.0",
     "d3-scale": "^3.2.3",
-    "d3-time": "^1.1.0"
+    "d3-time": "^2.1.1"
```

Comment:
I think with the dependency change, for CI we'll need to update the `yarn.lock` file in the monorepo root by running `yarn` there and committing the diff.

Reply:
Fixed - thanks :)